data/data/aws/variables-aws.tf

```diff
@@ -199,4 +199,10 @@ variable "aws_worker_iam_role_name" {
   type        = string
   description = "The name of the IAM role that will be attached to worker instances."
   default     = ""
-}
\ No newline at end of file
+}
+
+variable "aws_master_security_groups" {
+  type        = list(string)
+  description = "(optional) List of additional security group IDs to attach to the master nodes"
+  default     = null
```

Comment:
Changing the default to an empty list removes the need for coalescelist above since there will always be a list.

Reply:
Good idea. Fixed.